adoc/advanced_topics_proxy_quickstart.adoc

```diff
@@ -544,11 +545,11 @@ image::suma_proxy_old_details_page.png[]
 image::suma_proxy_old_details_spmigration.png[]
 
 . From this page you will see installed products listed on your proxy client, and the available target products.
-Select the wanted [guimenu]``Target Products``, in this case [systemitem]``SUSE Linux Enterprise Server 12 SP3`` with [systemitem]``SUSE Manager Proxy 3.2``.
+Select the wanted [guimenu]``Target Products``, in this case [systemitem]``SUSE Linux Enterprise Server 12 SP4`` with [systemitem]``SUSE Manager Proxy 3.2``.
```

Comment:
"Select the required [guimenu]``Target Products``. In this case, you will require ... "